src/libplctag/libplctag.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="libplctag.NativeImport" Version="0.0.5-alpha" />
+    <ProjectReference Include="..\libplctag.NativeImport\libplctag.NativeImport.csproj" />
```

Comment:
With the latest NativeImport (0.0.6-alpha) we won't need to reference this directly - and if we do change this the Nuget package won't work.